ledger-service/http-json/src/itlib/scala/http/AbstractHttpServiceIntegrationTest.scala

```diff
@@ -1508,7 +1510,7 @@ abstract class AbstractHttpServiceIntegrationTestTokenIndependent
           following: Seq[domain.Party] = Seq.empty,
       ): domain.CreateCommand[v.Record, domain.TemplateId.OptionalPkg] = {
         val followingList = lfToApi(
-          VAx.seq(VAx.partyStr).inj(domain.Party unsubst following)
+          VAx.seq(VAx.partyDomain).inj(following)
```

Comment:
I am not following why we need to change that. What is the relation with overloaded choices ?

Reply:
None, this is just a cleanup that I spotted when making various other changes.